src/config/globalHeaderData.tsx

```diff
@@ -52,14 +52,14 @@ export const navbarSublists = {
     },
     {
       name: 'CRDC Intake Applications',
-      link: '/#cgc',
+      link: '/#crdcia',
       className: 'navMobileSubItem',
     },
 ],
   About: [
     {
       name: 'Other Resources',
-      link: '',
+      link: 'oa',
```

Comment:
Is this intentionally a relative link? On the [questionnaire page](http://localhost:3000/questionnaire/1234) it links to `.../questionnaire/oa`

Reply:
Uhh, no. Was a typo. Should be /or my bad.